README.rst

```diff
@@ -0,0 +1,484 @@
+
+	Process type: kw_print_number_process
+	  Description: A Simple Kwiver Test Process
+	  Properties: _no_reentrant, _python
+	  Configuration:
+	    Name       : output
+	    Default    : .
+	    Description: The path for the output file.
+	    Tunable    : no
+
+	  Input ports:
+	    Name       : input
+	    Type       : integer
+	    Flags      : _required
+	    Description: Where numbers are read from.
+
+	  Output ports:
+
+As you can see, the process is very similar to the C++ `print_number`
+process.  As a result, the [".pipe" file is very
+similar](sprokit/pipelines/number_flow_python.pipe).
+
+In order to get around limitations imposed by the Python Global
+Interpreter Lock, we'll use a different Sprokit scheduler for this
+pipeline.  The `pythread_per_process` scheduler which does essentially
+what it says: it creates a Python thread for every process in the
+pipeline::
+
+	pipeline_runner -S pythread_per_process -p </path/to/kwiver/source>/sprokit/pipelines/number_flow_python.pipe>
+
+As with the previous pipeline, the numbers will be written to an output file, this time `numbers_from_python.txt`
+
+
+
+vital
+=====
+
+Vital is an open source C++ collection of libraries and tools that
+supply basic types and services to the Kitware KWIVER imagery tool
+kit.
+
+Overview of Directories
+-----------------------
+
+* CMake -- contains CMake helper scripts
+* tests -- contains testing related support code
+* vital -- contains the core library source and headers
+* vital/algo -- contains abstract algorithm definitions
+* vital/bindings -- contains C and Python bindings
+* vital/config -- contains configuration support code
+* vital/exceptions -- contains the exception class hierarchy
+* vital/io -- contains the classes that support reading and writing core data types
+* vital/kwiversys -- contains the code that supports the OS abstraction layer
+* vital/logger -- contains the classes that provide logging support
+* vital/plugin_loader --   contains the classes that provide plugin loading services
+* vital/tests -- contains the main testing code
+* vital/tools -- contains source for command line utilities
+* vital/types -- contains the source for the core data types
+* vital/util --   contains the source for general purpose utilities
+* vital/video_metadata -- contains the classes that support video metadata
+
+
+Running CMake
+-------------
+
+We recommend building kwiver out of the source directory to prevent
+mixing source files with compiled products.  Create a build directory
+in parallel with the kwiver source directory.  From the command line,
+enter the empty build directory and run::
+
+    $ ccmake /path/to/kwiver/source
+
+where the path above is the location of your kwiver source tree.  The
+ccmake tool allows for interactive selection of CMake options.
+Alternatively, using the CMake GUI you can set the source and build
+directories accordingly and press the "Configure" button.
+
+
+CMake Options
```

Comment:
This list  is missing ENABLE_ARROWS.

There is no discussion of arrow in this whole readme.